src/style.css

```
@@ -1 +1,44 @@
-/* add styles here */
+.page {
+  margin: 0;
+  padding: 0;
+}
+
+.page__body {
+  margin: 0;
+}
+
+.stars {
+  display: flex;
+}
+
+.stars__star {
+  width: 16px;
+  height: 16px;
+  background-image: url('../images/star.svg');
+  background-size: cover;
+  margin-right: 4px;
+}
+
+.stars--0 .stars__star:nth-child(-n + 0) {
+  background-image: url('../images/star.svg');
+}
+
+.stars--1 .stars__star:nth-child(-n + 1) {
+  background-image: url('../images/star-active.svg');
+}
+
+.stars--2 .stars__star:nth-child(-n + 2) {
+  background-image: url('../images/star-active.svg');
+}
```

```suggestion
.stars--1 .stars__star:nth-child(-n + 1),
.stars--2 .stars__star:nth-child(-n + 2) {
  background-image: url('../images/star-active.svg');
}
```
You can join selectors with the same styles like this

join all selectors